content/getting-started/php.textile

```diff
@@ -0,0 +1,207 @@
+
+# Install the Ably PHP SDK
+composer require ably/ably-php
+```
+
+<aside data-type='note'>
+<p>The code examples in this guide include a demo API key. If you wish to interact with the Ably CLI and view outputs within your Ably account, ensure that you replace them with your own API key.</p>
+</aside>
+
+
+h2(#step-1). Step 1: Configure the Ably Rest SDK client
+
+Clients establish a connection with Ably when they instantiate a REST SDK instance. This enables them to send messages across channels.
+
+* Create a @get_started.php@ file in your project and add the following code to instantiate the SDK and make requests to Ably. At the minimum you need to provide an authentication mechanism. Use an API key for simplicity, but you should use token authentication in a production app. A @client_id@ ensures the client is identified, which, with the PHP Rest SDK is identifying the name of the user sending a message:
+
+```[php]
+<?php
+
+require 'vendor/autoload.php';
+
+use Ably\AblyRest;
+
+$ably = new AblyRest(['key' => '{{API_KEY}}', 'clientId' => 'my-first-client']);
+```
+
+h2(#step-2). Step 2: Generate a JWT
+
+The Ably PHP SDK provides token authentication for secure client connections. Authentication with a JWT is recommended over using API keys directly in production because they can be scoped to specific capabilities and have expiration times. This is essential when authenticating frontend clients where embedding API keys would expose them publicly. As this is a PHP getting started guide, the API key would be considered securely stored as it is on the server side.
+
+This section will show you how to build a JWT with the Ably API credentials embedded, which you can then return to your frontend clients, for example from an API endpoint.
+
+First install the Firebase JWT library, which is used to create and verify JSON Web Tokens (JWTs):
+
+```[bash]
+composer require firebase/php-jwt
+```
+
+Import the necessary classes at the top of your @get_started.php@ file:
+
+```[php]
+use Firebase\JWT\JWT;
+use Firebase\JWT\Key;
+```
+
+Then create a new JWT with the following code:
+
+```[php]
+$currentTime = time();
+$payload = [
+    'iat' => $currentTime,
+    'exp' => $currentTime + 3600,
+    'x-ably-capability' => '{"*":["*"]}'
+];
+
+$apiKeyCredentials = explode(':', '{{API_KEY}}');
+$jwt = JWT::encode(
+    $payload,
+    $apiKeyCredentials[1],
+    'HS256',
+    $apiKeyCredentials[0]
+);
+
+echo $jwt;
+```
+
+h2(#step-3). Step 3: Publish a message to a channel
+
+Messages contain the data that a client is communicating, such as a short 'hello' from a colleague, or a financial update being broadcast to subscribers from a server. Ably uses channels to separate messages into different topics, so that clients only ever receive messages on the channels they are subscribed to. The PHP SDK and Rest API can only publish messages to channels and cannot subscribe to them. To subscribe to channels, for this guide, you will make use of the Ably CLI.
```

Comment:
I think we should drop mention of the REST API to be honest, as it's just another thing to learn in a getting started. 

(also another nitpick on "for this guide" as it makes it sound like you'd read multiple)